prime-router/settings/organizations-prod.yml

```diff
@@ -36,9 +36,9 @@
       deidentify: false
```

Comment:
Please double-check my changes here super carefully!    I've changed all the timezones to EASTERN, for our own sanity. 

For the sends that go 12 times a day, the idea is that we send 15 minutes after SR runs.
For the sends that go once a day, the idea is we bunch them late at night (midnight for redox) or early morning, so there is (theoretically) a window **just after 9:15am ET,** when the system might be free of Work-In-Progress.  Note:   I have also added a report into Metabase, that lists WIP, so we now have an easy way to check this.

Did I get this right?